docker/docker/Dockerfile.linux.arm

```diff
@@ -1,4 +1,4 @@
-FROM arm32v6/docker:19.03.8-dind
+FROM arm32v6/docker:20.10.7-dind
```

Comment:
Reverting back to 19.03.8 for arm32 should allow us to move forward with 20.10 on rest of platforms while keeping (on life support, as it seems Docker very much does not care about that platform anymore) arm32 usable

https://github.com/docker-library/docker/issues/260
https://github.com/drone-plugins/drone-docker/pull/306